.github/workflows/Container-Ubuntu-22.yml

```diff
@@ -0,0 +1,28 @@
+# GitHub Action Workflow for building the Ubuntu 22 images.
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+name: "Ubuntu 22 Containers"
+
+# This workflow only runs (on the main branch or on PRs targeted
+# at the main branch) and if files inside the Ubuntu-22 directory
+# have been modifed/added/removed...
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+    paths:
+      - 'Containers/Ubuntu-22/**'
+  pull_request:

```

Comment:
Are you sure that this needs a `pull_request` trigger?

Since the `pull_request` trigger runs on open, synchronize, and reopened types by default and this is a public repo that could lead to a lot of images potentially being created that are not useful.

Reply:
I see the `if: ${{ github.ref == 'refs/heads/main' }}` push condition now. That's fine.